src/documentReview/Preview.vue

```diff
@@ -60,7 +60,22 @@
                 :selectedException="docData.fairOpportunity.exception_to_fair_opportunity"
               />
             </li>
-          </ol>          
+          </ol>
+
+          <IncumbentContractorName
+              :isForm="isForm"
+              :value.sync="docData.currentContract.incumbent_contractor_name"
+          />
+
+          <ContractNumber
+              :isForm="isForm"
+              :value.sync="docData.currentContract.contract_number"
+          />
+
+          <TaskOrderNumber
+              :isForm="isForm"
+              :value.sync="docData.currentContract.task_delivery_order_number"
+          />
```

Comment:
Please move this to under "Do you have a current contract..." `li`
![Screen Shot 2022-09-23 at 9 57 39 AM](https://user-images.githubusercontent.com/90333349/191978243-27a92101-80ef-4185-b8ed-4b9bedf5ceff.png)